src/app/charts/chart.component.ts

```diff
@@ -62,7 +62,7 @@ export class ChartComponent implements OnChanges {
     }
 
     onExportClicked() {
-        this.csvService.downloadAsCSV(this.chartData);
+        this.dataExportService.downloadAsJSON(this.chartData);
```

Comment:
`chartData` has undergone some transformation since the raw response from the API request to format for drawing the line graphs. It is similar, but different. Since my understanding is we are trying to mirror the API I think we might want the raw response data.

https://github.com/azavea/climate-change-lab/blob/develop/src/app/charts/chart.component.ts#L57

Thoughts @CloudNiner ?